.github/workflows/release.yml

```diff
@@ -19,16 +19,15 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v4
 
-    - name: Set up Python 3
-      uses: actions/setup-python@v5
+    - name: Install Poetry
+      uses: snok/install-poetry@v1
       with:
-        python-version: '3.x'
+          virtualenvs-create: true
+          virtualenvs-in-project: true
 
     - name: Install dependencies & build
       run: |
-        pip install -U pip
-        pip install build
-        python -m build
+        poetry build -o dist --no-interaction --no-cache --verbose
```

Comment:
You dont need to do this sort of builds.

You can just call `poetry publish --build` and the build and upload will be done by poetry.

https://github.com/baseplate-admin/django-ltree-2/blob/55584e9075cad1c9f4a0141523048630a52a4121/.github/workflows/publish.yml#L6-L26

Reply:
I was attempting to use the repository trust feature of PyPI, however if poetry has this built-in, I'll implement it with a token like before.

Reply:
Update: Resolved with the https://github.com/hitblast/avro.py/commit/e1563190dc37c66fd81b2fb069357a46149dd38f commit, needs a test run first